imports/ui/report-wizard/report-wizard.jsx

```diff
@@ -69,16 +70,26 @@ class ReportWizard extends Component {
         dispatch(replace(match.url.replace(viewMode, enforcedViewMode)))
       }
     }
+    if (prevProps.reportItem !== reportItem) {
+      this.setState({isEditable: false})
```

Comment:
It would, but it might do it on other occasions when you didn't intend for it to happen... Just move to `handleSubmit` instead of here, it's not the right place to put it

Reply:
I can move setState to `handleSubmit`. One caveat is that it would display the old title temporarily until the new title appears. The current code renders the title only when the title is updated.

Reply:
Do move this to `handleSubmit`. The way it is now causes erratic behavior while changing other fields (try editing the title while uploading a picture, or change "additional comments" while the field is editable).
It also causes the save button to respond in delay which is not good UX.
To avoid the effect of the new name switching back to the old name briefly, store the current name in a state variable which will be used for editing and presentation instead of the prop variable,  and re-set the state variable to the prop variable's value in `componentDidUpdate` if `(!prevProps.reportItem || prevProps.reportItem.title !== this.props.reportItem.title)`